Makefile

```diff
@@ -25,7 +25,7 @@ OBJ := $(SRC:.c=.o)
 
 # define default flags, and override to append mandatory flags
 ARFLAGS := rcs
-CFLAGS ?= -O3 -Wall -Wextra -Wshadow -pedantic
+CFLAGS ?= -O3 -Wall -Wextra -Wshadow -Wno-unused-parameter -pedantic
```

Comment:
You can just remove the parameter from `ts_parser__call_keyword_lex_fn`, seems like it's no longer used.